app/views/boards/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<body>
+ <div class="container">
+ <table class="block-center"
+   <table class="one">
```

Comment:
You can nest tables by putting one table inside a cell (the `<td></td>` portion) belonging to another table, but the current approach will not work and will actually result in the two getting rendered one after the other. The reason you're not seeing this is because you didn't close the table tag on the preceding line and the browser is treating them as one element.

The solution is just to drop one of the `<table></table>` pairs. You only need one to make this work.